specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-01-02-preview/managedClusters.json

```diff
@@ -2823,6 +2829,10 @@
         "capacityReservationGroupID": {
           "$ref": "#/definitions/CapacityReservationGroupID",
           "description": "AKS will associate the specified agent pool with the Capacity Reservation Group."
+        },
+        "hostGroupID": {
+          "$ref": "#/definitions/HostGroupID",
+          "description": "The ID for Dedicated Host Group."
```

Comment:
I feel that this documentation is missing a little bit:
1. What is dedicated host group _for_? Do we have a link to a blog or article discussing it we could put here? (see other comments with links for an example of how this can be done).
2. What is the expected format of this ID? Is it an ARM ID? Say something like: "This is of the form: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/subnets/{subnetName}". See existing properties for examples of that as well
3. Are there restrictions on when this can be set? Only at creation? If there are no restrictions then what you have is great. If there are restrictions then we should document those as well.
4. If doing the above gets too long, you can split this across `title` and `description` (see other properties for example).

Reply:
The result might look something like this:

```
"title": "The ID of the Dedicated Host Group to provision virtual machines from"
"description": "This is of the form: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}. Once set at agent pool creation, this cannot be changed. For more information see [Azure dedicated hosts](https://docs.microsoft.com/azure/virtual-machines/dedicated-hosts)"
```

Reply:
Add more details